lib/reek/smells/control_parameter.rb

```diff
@@ -42,6 +42,8 @@ module Smells
     # the source code.
     #
     # See {file:docs/Control-Parameter.md} for details.
+    #
+    # :reek:InstanceVariableAssumption
```

Comment:
Due to the changes in the detector, this class no longer has this smell so these lines can be removed.